operations/network/management_network/Management_Network_Access_Port_Configurations.md

```diff
@@ -106,14 +106,14 @@ interface lag 17 multi-chassis
 ## Gigabyte/Intel NCN Worker Port Configuration
 The cabling guidelines for all servers can be found in [Cable Management Network Servers](../../../install/cable_management_network_servers.md).
 
-Mellanox port configuration:
+Aruba port configuration:
```

Comment:
Customers should never have an Aruba combined with Gigabyte/Intel.

Reply:
Correccted in db050a52